frontend/src/util/worksheet_utils.js

```diff
@@ -70,6 +70,9 @@ export function renderSize(size, includeBytes = false) {
     // size: number of bytes
     // includeBytes: whether or not to include 'bytes' string in the return value
     // Return a human-readable string.
+    if (!size) {
```

Comment:
Be careful about supporting 0 properly

Reply:
Not supporting `0` was intentional 👀 e.g. in the bundle row instead of showing `0` we just leave it blank. And in the detail sidebar, instead of showing `0` we show `<none>`